src/conversion-settings.ts

```diff
@@ -31,32 +33,16 @@ export interface ConversionSettings {
   readonly namespaces: Set<string>;
 
   /**
-   * Files to exclude from conversion (ie lib/utils/boot.html).
+   * A map from original package names to entrypoints to be used when converting
+   * each listed package. Setting an entry in this map completely replaces the
+   * default entrypoints for that package.
    */
-  readonly excludes: Set<string>;
+  readonly entrypoints: Map<string, OriginalDocumentUrl[]>;
```

Comment:
since this is a map, I think it would benefit from a more descriptive name. How about `packageEntrypoints` to indicate that this is a map of package-based entrypoints?

Reply:
Updated to `packageEntrypoints`.